packages/http-api/src/abstract-http-client.ts

```diff
@@ -6,39 +6,44 @@ import {
     IHttpRequest,
     IHttpResponseMessage,
     IHttpRequestMessage,
-    IHttpResponseRejectMessage
+    IHttpResponseRejectMessage,
+    IQueue,
 } from '@testring/types';
 import { loggerClient } from '@testring/logger';
 import { HttpCookieJar } from './cookie-jar';
+import { Queue } from '@testring/utils';
 
 const nanoid = require('nanoid');
 
 const toString = c => c.toString();
 
 export abstract class AbstractHttpClient implements IHttpClient {
     protected abstract broadcast(options: IHttpRequestMessage): void;
+    private queue: IQueue<Function>;
+    private queueRunning = false;
 
-    constructor(protected transportInstance: ITransport) {
+    constructor(protected transportInstance: ITransport, private httpThrottle: number) {
```

Comment:
Maybe better use options object, with httpThrottle property to solve problem possible additional properties in feature.